src/Microsoft.Azure.SignalR.Management/ServiceManagerOptions.cs

```diff
@@ -25,30 +23,72 @@ public class ServiceManagerOptions
 
         /// <summary>
         /// Gets or sets the connection string of Azure SignalR Service instance.
+        /// Gets or sets the connection string of Azure SignalR Service instance and switches to single-endpoint mode.
```

Comment:
remove line 25

Reply:
don't use `mode` for single/multiple service manager. Since we already use `mode` to describe `Transient` and `Persistent`.